kahuna/public/js/image/view.html

```diff
@@ -120,104 +120,102 @@ <h2 class="image-info__title">
-                      onbeforesave="ctrl.updateMetadataField('credit', $data)"
-                      e:ng-class="{'image-info__editor--error': $error, 'image-info__editor--saving': creditEditForm.$waiting}"
-                      e:form="creditEditForm">
-
-                    <span ng:if="ctrl.metadata.credit">
-                        <a ui:sref="search.results({query: (ctrl.metadata.credit | queryFilter:'credit')})">{{ctrl.metadata.credit}}</a>
-                    </span>
-                    <span ng:if="! ctrl.metadata.credit">
-                        unknown (click ✎ to add the credit)
-                    </span>
-                </span>
-            </div>
-
-            <div ng:if="ctrl.metadata.subLocation || ctrl.metadata.city || ctrl.metadata.state || ctrl.metadata.country"
-                 class="metadata-line">
-                in
-                <span ng:repeat="prop in ['subLocation', 'city', 'state', 'country']" ng:if="ctrl.metadata[prop]">
-                    <span class="metadata-line__info">
-                        <a ui:sref="search.results({query: (ctrl.metadata[prop] | queryFilter:ctrl.locationFieldMap[prop])})">{{ctrl.metadata[prop]}}</a><!-- no whitespace before comma
-                                                                                                                                                           --></span><span ng:if="! $last">,</span>
-                </span>
-            </div>
-
-            <div ng:if="ctrl.metadata.copyright"
-                 class="metadata-line">
-                ©
-                <span class="metadata-line__info">
-                    <a ui:sref="search.results({query: (ctrl.metadata.copyright | queryFilter:'copyright')})">{{ctrl.metadata.copyright}}</a>
-                </span>
-                <!-- FIXME: do we want copyright or copyrightNotice? -->
-            </div>
-
-            <div class="metadata-line">
-                uploaded
-                <span class="metadata-line__info">
-                    {{ctrl.image.data.uploadTime | date:'d MMM yyyy, HH:mm'}}
-                </span>
-                <span class="metadata-line__info">
-                    (<a ui:sref="search.results({query: (ctrl.image.data.uploadedBy | queryFilter:'uploader')})">{{ctrl.image.data.uploadedBy | stripEmailDomain}}</a>)
-                </span>
-            </div>
-        </div>
-
-        <div class="image-info__group" ng:if="!ctrl.crop">
-            <div class="metadata-line">
-                dimensions
-                <span class="metadata-line__info">{{ctrl.dimensions.width}} &times; {{ctrl.dimensions.height}}</span>
-            </div>
-        </div>
-
-        <div class="image-info__group" ng:if="ctrl.crop">
-            <div class="metadata-line">
-                dimensions (crop)
-                <span class="metadata-line__info">{{ctrl.dimensions.width}} &times; {{ctrl.dimensions.height}}</span>
-            </div>
+            <table>
```

Comment:
We talked about that and in brief, yes, but it's also less convenient than tables because table columns automatically adapt to their widest cell, which you can't do with `dl` (you have to hardcode the width). We suggested to go with the simpler table for now to fix the visual style and move to dl as a future step.